data/jobs.txt

```diff
@@ -2197,6 +2197,27 @@ mission "Colonists [1]"
 		payment 10000
 		dialog "The colonists depart your ship after paying you <payment>."
 
+mission "Stranded Field Trip [0]"
+	name "Stranded field trip to <planet>"
+	job
+	repeat
+	description `These <bunks> school students have been stranded on an educational field trip due to the unexpected bankruptcy of their booked transport. They have the supplies to continue their studies but need to be safely returned home at some point. The school & parents have put up a reward of <payment>.`
+	passengers 20 100 .6
+	cargo "educational supplies" 2 10 .6
+	to offer
+		random < 10
+		"passenger space" > 50
+	source
+		attributes "near earth" "deep" "paradise" "tourism"
```

Comment:
Please include a `government` filter in the source and destination for these new jobs, as some of these attributes are used outside of human space.